tfhe/src/high_level_api/integers/types/base.rs

```diff
@@ -382,60 +383,8 @@ where
 }
 
 macro_rules! generic_integer_impl_operation (
-    ($rust_trait_name:ident($rust_trait_method:ident,$op:tt) => $key_method:ident) => {
-        #[doc = concat!(" Allows using the `", stringify!($op), "` operator between a")]
-        #[doc = " `GenericInteger` and a `GenericInteger` or a `&GenericInteger`"]
```

Comment:
u8 does not impl shift by a u8 amount nor Wrapping<u8> only by u32/usize, the macros generated docs for binary ops
where both lhs and rhs were Wrapping<u8>. So i prefer removing it and rely on the user docs to showcase operations. 

If having this kind of docstring is that important then I think its better to re-craft a better one, that would also be able to
generate docs for assign and scalar ops

Reply:
Understood

Care to open a follow-up issue so that we add this back in down the line ? Or maybe have some sort of "stub" documentation pointing to the rust documentation for the expected behavior ?

Reply:
@tmontaigu maybe repurpose 97 ? up to you

Reply:
agreed